src/metabase/api/dashboard.clj

```diff
@@ -242,6 +244,7 @@
                            ;; Get cards from existing dashboard and associate to copied dashboard
                            (doseq [card (:ordered_cards existing-dashboard)]
                              (api/check-500 (dashboard/add-dashcard! <> (:card_id card) card)))))]
+    (analytics/track-event :dashboard_created api/*current-user-id* {:dashboard_id (u/the-id dashboard)})
```

Comment:
I'm pretty sure `track-event` is a side effect. Calling `track-event` 20 times is not the same as calling it once... definitely not safe to do inside an STM transaction or some other situation where we don't know how many times it might be called. Should this be called `track-event!`?